app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -57,7 +57,22 @@ message Target {
     // Name of the LUCI recipe to use for the builder.
     optional string recipe = 13;
     // Additional tag for the target.
-    repeated string tags = 14;
+    optional Tags tags = 14;
+}
+
+// Tags for a specific infrastructure task
+// Next ID: 6
+message Tags {
```

Comment:
Based on [builds_service.proto](https://source.chromium.org/chromium/infra/infra/+/main:go/src/go.chromium.org/luci/buildbucket/proto/builds_service.proto;l=280;drc=adbd1e588d722858df732b1ecb23e09c00ab1b54), can we make this a StringPair as well?

Reply:
sounds good

Reply:
it looks like StringPair is a custom structure defined in https://source.chromium.org/chromium/infra/infra/+/main:luci/appengine/swarming/proto/api/swarming.proto?q=StringPair&ss=chromium%2Finfra%2Finfra

I will need to duplicate the code I assume?

Reply:
Yeah, easier to copy it here since it's relatively simple